libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -143,15 +143,22 @@ const AP_Param::GroupInfo AP_BattMonitor::var_info[] = {
 
 #endif // AP_BATT_MONITOR_MAX_INSTANCES > 1
 
-    // @Param: _VOLT_TIMER
+    // @Param: _LOW_TIMER
     // @DisplayName: Low voltage timeout
     // @Description: This is the timeout in seconds before a low voltage event will be triggered. For aircraft with low C batteries it may be necessary to raise this in order to cope with low voltage on long takeoffs. A value of zero disables low voltage errors.
     // @Units: s
     // @Increment: 1
     // @Range: 0 120
     // @User: Advanced
-    AP_GROUPINFO("_VOLT_TIMER", 21, AP_BattMonitor, _volt_timeout, AP_BATT_LOW_VOLT_TIMEOUT_DEFAULT),
-        
+    AP_GROUPINFO("_LOW_TIMER", 21, AP_BattMonitor, _low_voltage_timeout, AP_BATT_LOW_VOLT_TIMEOUT_DEFAULT),
+
+    // @Param: _LOW_TYPE
+    // @DisplayName: Low voltage type
+    // @Description: Voltage type used for detection of low voltage event
+    // @Values: 0:Raw Voltage, 1:Sag Compensated Voltage
+    // @User: Advanced
+    AP_GROUPINFO("_LOW_TYPE", 22, AP_BattMonitor, _low_voltage_source, 0),
```

Comment:
You have #define's used at the moment to do the switch later, and while I'd rather see them as an enumerated type this should be using one of the two instead.